modules/geotiff/package.json

```diff
@@ -32,18 +32,15 @@
     "pre-build": "npm run build-bundle",
     "build-bundle": "webpack --display=minimal --config ../../scripts/webpack/bundle.js"
   },
+  "dependencies-disabled": {
+    "geotiff": "ilan-gold/geotiff.js#ilan-gold/viv_094"
+  },
```

Comment:
Any docs on this? I can't seem to find any on a quick search

Reply:
Docs on what? I just moved it to an unused key - think of it as a comment.

Reply:
Ah I thought `dependencies-disabled` had a specific meaning in `package.json`, something like installing the dependencies for upstream `geotiff` below but then replacing the `geotiff` code with Ilan's fork.